jwala-services/src/main/java/com/cerner/jwala/service/webserver/impl/WebServerServiceImpl.java

```diff
@@ -139,6 +139,12 @@ public WebServer updateWebServer(final UpdateWebServerRequest anUpdateWebServerC
                                      final User anUpdatingUser) {
         anUpdateWebServerCommand.validate();
 
+        WebServer originalWebServer=getWebServer(anUpdateWebServerCommand.getId());
+        if(!anUpdateWebServerCommand.getNewName().equalsIgnoreCase(originalWebServer.getName())&& !anUpdateWebServerCommand.getState().toStateLabel().equals(WebServerReachableState.WS_NEW.toStateLabel())){
```

Comment:
Please change Enum comparison to
    
    !WebServerReachableState.WS_NEW.equals(anUpdateWebServerCommand.getState())

Reason being: Enums can be compared by equals as is and it's best practice to compare using the constant first before the value to prevent NPEs e.g. "Jedd".equalsIgnoreCase(employee.firstName)